scripts/qa_crowbarsetup.sh

```diff
@@ -3208,6 +3208,11 @@ function oncontroller_manila_generic_driver_setup()
     openstack ip floating add $manila_tenant_vm_ip manila-service
 
     [ $? != 0 ] && complain 44 "adding a floating ip to the manila service VM failed"
+
+    # check that the service VM is pingable. otherwise manila tempest tests will fail later
+    wait_for 300 1 "ping -q -c 1 -w 1 $manila_tenant_vm_ip >/dev/null" \
```

Comment:
@dirkmueller there is no race. It's just about warning that the tempest test will fail (but there's plenty of time between this ping test and starting the tempest run)